march_hardware/src/PDOmap.cpp

```diff
@@ -21,6 +21,7 @@ std::unordered_map<IMCObjectName, IMCObject> PDOmap::all_objects = {
   { IMCObjectName::TargetPosition, IMCObject(0x607A, 32) },
   { IMCObjectName::TargetTorque, IMCObject(0x6071, 16) },
   { IMCObjectName::QuickStopDeceleration, IMCObject(0x6085, 32) },
+  { IMCObjectName::MotorVoltage, IMCObject(0x2066, 32) },
```

Comment:
I believe 2066 is an SDO object, which doesn't need to be mapped.

Reply:
It would be neater if we could add this to the PDO, so maybe find out if this is possible through testing?

Reply:
Actually, this register cannot be mapped at all according to the IMC CoE manual. (see page 236)

Reply:
I realized this, so I think it's better to remove the register from the PDO mapping and use the read function Rogier will make